### ISSUE TITLE
[Question] Can we reduce the size of the output_dir ?

### ISSUE THREAD

**Issue by simepy** [ThilinaRajapakse/simpletransformers#410]
Hi everyone  !

The size of my "output_dir" is ~700mb, he contain the last epoch of BERT fine-tune model
He contain:
```
1,0K	config.json
4,0K	model_args.json
1,4G	optimizer.pt
679M	pytorch_model.bin
512    scheduler.pt
512    special_tokens_map.json
512    tokenizer_config.json
4,0K	training_args.bin
976K	vocab.txt
```
How can I reduce this size ?
Can I remove some things inside without damage the performance of my model ?

Thank you in advance

**Comment by ThilinaRajapakse** [comment]
You can remove the optimizer and scheduler if you don't plan on retraining from a given save. They are only needed to resume training.

**Comment by simepy** [comment]
Ok, perfect thanks @ThilinaRajapakse :+1:
